aws_lambda_builders/workflows/nodejs_npm/npm.py

```diff
@@ -0,0 +1,49 @@
+"""
+Wrapper around calling npm through a subprocess.
+"""
+
+import logging
+
+from aws_lambda_builders.exceptions import LambdaBuilderError
+
+from .utils import OSUtils
+
+LOG = logging.getLogger(__name__)
+
+
+class NpmExecutionError(LambdaBuilderError):
```

Comment:
actually, you misunderstood my previous comment. Don't inherit directly from LambdaBuilderError. Instead just follow the pattern of having your custom base error class that does the interpolation, and subclasses just set the MESSAGE attribute.

In this case, since you basically have only one exception, you can consider this to be the base error. Something like this:

```
class NpmExecutionError(Exception):
  MESSAGE = "NPM Failed: {message}"
  def __init__(self, **kwargs):
             Exception.__init__(self, self.MESSAGE.format(**kwargs))
```

Reply:
ok, done this in https://github.com/gojko/aws-lambda-builders/commit/d141caf9b61c39d2aeab763367921b15a65c2568